### DISCUSSION TITLE
Enhancement: lag problem throughout the game

### DISCUSSION THREAD

**Discussion by MRbaldi87** [FunkinCrew/Funkin#1675] [Ideas]
good I just come to give a problem that I have with the game is that look,
I don't know if it's the game or the pc (I hope it's the pc that the game gives constant lageos or slowness throughout the game and that doesn't happen in the browser version if not in the downloadable version, I would like to at least know what things you need The pc to play well without those annoying lageos or slowness that there is, please, the game is good and I would like to play it but continue like this, invite me to enjoy it well

**Comment by ghost** [comment]
The game doesn't run well on low-end PCs because of HaxeFlixel's asset loading method. On a relatively new or high-end PC, it should run fine.
